io.httpie.HTTPie.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.httpie.HTTPie</id>
+  <launchable type="desktop-id">io.httpie.HTTPie.desktop</launchable>
+  <name>HTTPie</name>
+  <developer_name>HTTPie</developer_name>
+  <summary>Cross-platform API testing client for Humans</summary>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
I'm unable to find any mentions of the actual licence for this app.

Reply:
Author plans to Open Source it in Future. But, right now,it's proprietary.

https://github.com/httpie/desktop/issues/32